client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
We've been following [the approach mentioned by Ian Obermiller](https://codemebaby.wordpress.com/2015/07/07/facebook-engineer-ian-obermiller-breaks/) for some time now in Spectrum and overall it works pretty well. If you look at our [list of actions](https://github.com/Automattic/wp-calypso/blob/master/client/lib/upgrades/constants.js), you'll see that it's very easy to scan. I think ordering and prefixing is important. I agree that we shouldn't prefix by area though - in our case we use the name of the data whose action is performed on.

Using `REQUEST` instead of `FETCH` seems like a good idea to me. We used `SET` in some cases as it made more sense. However we should probably try to limit these variants in some ways for consistency purpose.